src/GovUk.Education.ExploreEducationStatistics.Common/Model/Timestamps.cs

```diff
@@ -0,0 +1,24 @@
+#nullable enable
+using System;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Model
+{
+    public interface ICreatedTimestamp
+    {
+        DateTime? Created { get; set; }
```

Comment:
I was wondering how you would solve the problem of some entities having a not-nullable Created field in the database.

For newly added types I think we would like this field to be not-null in the database.

If this interface is applied to any existing types with a `Created` field that is not optional does it cause a database migration converting the field from not-null to allow null?

Reply:
As discussed on Slack, this has been fixed by adding type parameters to the interfaces to let the consumers decide the type of their timestamp field. This is also more flexible as it allows them to use different types like `DateTimeOffset` if required.

Reply:
Cool, changes to allow different types of Date look good!